ripple/src/main/webapp/businessdetails.js

```diff
@@ -368,17 +366,29 @@ function bdOnload() {
         }
       }
   getDocByDocId("businessDetails", businessId, lambda);
-
-  var lambda2 = (doc) => {
+  
+  /* Function to iterate over all the tags for a business and add them to the business details page */
+  var queryTags = (doc) => {
         if (doc.exists) {
+          var tagsMap = doc.data().tags;
+          var tagsList = Object.keys(tagsMap);
           var j;
-          for (j = 0; j < 4; j++) {
-            console.log("tags: " + Object.keys(doc.data().tags).toString());
-            //createCarouselElement(doc.data().galleryBlobKeys[i], "imageGallery");
+          for (j = 0; j < tagsList.length; j++) {
+            var tag = tagsList[j];
+            //console.log(tag);
```

Comment:
just double checking this is intentional

Reply:
Wait are you referring to the console.log statement? I plan to remove all of them when I am done testing everything!